internal/audit/conf.go

```diff
@@ -3,17 +3,58 @@
 
 package audit
 
+import (
+	"fmt"
+
+	"gopkg.in/yaml.v2"
```

Comment:
The project already uses `gopkg.in/yaml.v3` indirectly. What are your thoughts about v3 vs v2?

Reply:
The config library (https://github.com/uber-go/config) uses `yaml.v2` , hence why I stuck to using `yaml.v2` here as well.